### PR TITLE
Add version command-line option

### LINKED ISSUE #94: WISH: `--version` to display version of vcf_* tools
First timer here (thxs!). Please consider adding command-line option `--version`, e.g.
```sh
$ vcf_validator --version
0.6
```
or possible have `--help` output the version.

### PR DESCRIPTION
**Pull request by Anishka0107**
Fixes #94

### CODE REVIEW

File: src/debugulator_main.cpp [hunk 1]
```diff
@@ -55,6 +57,11 @@ namespace
           return -1;
       }
 
+      if (vm.count(ebi::vcf::VERSION)) {
+          std::cout << "VCF-Debugulator version " << VERSION_MAJOR << "." << VERSION_MINOR << std::endl;
```
Comment (cyenyxe) [inline src/debugulator_main.cpp]: `vcf-debugulator` to match the capitalization of the binary? (same for `vcf-validator`)

File: cmake_config.hpp.in [hunk 2]
```diff
@@ -0,0 +1,2 @@
+#define VERSION_MAJOR @vcf-validator_VERSION_MAJOR@
```
Comment (cyenyxe) [inline cmake_config.hpp.in]: Can this file be placed inside the `inc` folder or wouldn't it work that way?

Comment (Anishka0107) [reply cmake_config.hpp.in]: done in next commit

Comment (jmmut) [reply cmake_config.hpp.in]: @cyenyxe, did you mean only the cmake_config.hpp.in? or the cmake_config.hpp as well?

Comment (cyenyxe) [reply cmake_config.hpp.in]: I am fine with tracking only the .in in the repository as the other auto-generated, but I think both files should be placed inside `inc` for consistency.

Comment (Anishka0107) [reply cmake_config.hpp.in]: fixed in next commit

File: src/debugulator_main.cpp [hunk 3]
```diff
@@ -55,6 +57,11 @@ namespace
           return -1;
       }
 
+      if (vm.count(ebi::vcf::VERSION)) {
+          std::cout << "vcf-debugulator version " << VERSION_MAJOR << "." << VERSION_MINOR << std::endl;
```
Comment (cyenyxe) [inline src/debugulator_main.cpp]: Could these messages be printed at the beginning of the help section too? Something like this:

```
vcf-debugulator version 0.6

Usage: vcf-debugulator [OPTIONS] [< input_file]
Allowed options:
  -h [ --help ]                 Display this help
  -v [ --version ]              Display version of the debugulator
  -i [ --input ] arg (=stdin)   Path to the input VCF file, or stdin
  -e [ --errors ] arg           Path to the errors report from the input VCF 
                                file
  -l [ --level ] arg (=warning) Validation level (error, warning, stop)
  -o [ --output ] arg (=stdout) Write to a file or stdout
```

**Pull request merged**